larq/callbacks.py

```diff
@@ -24,6 +24,8 @@ class HyperparameterScheduler(tf.keras.callbacks.Callback):
     schedule: a function that takes an epoch index as input
         (integer, indexed from 0) and returns a new hyperparameter as output.
     hyperparameter: str. the name of the hyperparameter to be scheduled.
+    unit: str (optional), what interval unit to change the hyperparameter at. Can be
+        either "epoch" (default) or "step".
```

Comment:
I think `interval` would be a better name for this parameter.

Reply:
The [TensorBoardCallback](https://keras.io/callbacks/#tensorboard) has an `update_freq` which can be either `'batch'` or `'epoch'` or integer. I think it would be good to align our selfs with a similar naming scheme.